storage/migrations/01_oasis_3_consensus.up.sql

```diff
@@ -74,6 +74,9 @@ CREATE TABLE oasis_3.events
   FOREIGN KEY (tx_block, tx_index) REFERENCES oasis_3.transactions(block, tx_index) DEFERRABLE INITIALLY DEFERRED
 );
 CREATE INDEX ix_events_related_accounts ON oasis_3.events USING gin(related_accounts);
+CREATE INDEX ix_events_tx_block ON oasis_3.events (tx_block);  -- for fetching events without filters
+CREATE INDEX ix_events_tx_hash ON oasis_3.events (tx_hash);
```

Comment:
should be joining through the transactions table and looking up events by tx_block, tx_index. do we ever not?